test/v2_integration.py

```diff
@@ -1178,36 +1161,42 @@ def test_new_order_policy_errs():
     # subproblems are properly represented.
     ok = False
     try:
-        order = client.new_order(csr_pem)
+        client.new_order(csr_pem)
     except messages.Error as e:
         ok = True
         if e.typ != "urn:ietf:params:acme:error:rejectedIdentifier":
-            raise(Exception("Expected rejectedIdentifier type problem, got {0}".format(e.typ)))
-        if e.detail != 'Error creating new order :: Cannot issue for "between-addr.in-addr.arpa": The ACME server refuses to issue a certificate for this domain name, because it is forbidden by policy (and 1 more problems. Refer to sub-problems for more information.)':
-            raise(Exception("Order problem detail did not match expected"))
+            raise (Exception("Expected rejectedIdentifier type problem, got {0}".format(e.typ)))
+        if (
+            e.detail != 'Error creating new order :: Cannot issue for "between-addr.in-addr.arpa": The ACME server refuses to '
+            "issue a certificate for this domain name, because it is forbidden by policy (and 1 more problems. "
+            "Refer to sub-problems for more information.)"
```

Comment:
a) this now has a mixture of single- and double-quotes; and
b) let's not use python's implicit multiline string contatenation. Instead, wrap the RHS of this inequality in parens, and explicitly concatenate the lines, imo.

Reply:
I can explicitly wrap and concat, but black style is to prefer double quotes, but it prefers avoiding escaping more, which is a bit of a problem here.